ui/src/frontend/notes_panel.ts

```diff
@@ -82,11 +85,33 @@ export class NotesPanel extends Panel {
         '.notes-panel',
         {
           onclick: (e: PerfettoMouseEvent) => {
-            this.onClick(e.layerX - TRACK_SHELL_WIDTH, e.layerY);
+            this.onClick(e.layerX - globals.state.trackShellWidth, e.layerY);
             e.stopPropagation();
           },
+          onmousemove: (e: PerfettoMouseEvent)=>{
+            if (e.currentTarget instanceof HTMLElement &&
+              (
+                (e.layerX +2) >= globals.state.trackShellWidth &&
```

Comment:
The ±2 buffer should maybe be extracted to a constant? I think it counts as a "magic number."
At the very least, without whitespace between the sign and the number it looks like there isn't a mathematical operator in this expression. Especially in the -2 case.

Same comment everywhere this code is copied.